### PR TITLE
Added: Wrapper for lrx-proc -m

### PR DESCRIPTION
**Pull request by Vaydheesh**
- Fixed: Windows tempfile issue
- Added: wrapper for lrx-proc -m

### CODE REVIEW

File: build-swig-wrapper.sh [hunk 1]
```diff
@@ -7,3 +7,9 @@ cd lttoolbox
 ./autogen.sh --enable-python-bindings && make
 cd python
 python3 setup.py install
+
+git clone -b swig_wrapper https://github.com/Vaydheesh/apertium-lex-tools
```
Comment (sushain97) [inline build-swig-wrapper.sh]: Can we switch the `lttoolbox` above to be apertium's now that it's been merged?

Comment (sushain97) [reply build-swig-wrapper.sh]: Also, add the `.git` here :)

File: apertium/utils.py [hunk 2]
```diff
@@ -36,23 +38,35 @@ def execute_pipeline(inp: str, commands: List[List[str]]) -> str:
     """
     end = inp.encode()
     for command in commands:
+        lttoolbox.LtLocale.tryToSetLocale()
```
Comment (sushain97) [inline apertium/utils.py]: Should this go inside the `lt-proc` branch?

File: apertium/utils.py [hunk 3]
```diff
@@ -36,23 +38,35 @@ def execute_pipeline(inp: str, commands: List[List[str]]) -> str:
...
+        input_file = tempfile.NamedTemporaryFile('w', delete=False)
+        output_file = tempfile.NamedTemporaryFile('r', delete=False)
+        arg = command[1][1] if len(command) == 3 else ''
+        path = command[-1]
+        wrapper_flag = True
```
Comment (sushain97) [inline apertium/utils.py]:
```suggestion
        used_wrapper = True
```

Generally, there's a better variable name (than one that says "flag") when you're trying to use a flag.

File: apertium/utils.py [hunk 4]
```diff
@@ -36,23 +38,35 @@ def execute_pipeline(inp: str, commands: List[List[str]]) -> str:
     """
     end = inp.encode()
     for command in commands:
+        lttoolbox.LtLocale.tryToSetLocale()
+        input_file = tempfile.NamedTemporaryFile('w', delete=False)
+        output_file = tempfile.NamedTemporaryFile('r', delete=False)
```
Comment (sushain97) [inline apertium/utils.py]: Please add a comment re. why we do this (i.e. for Windows).

File: apertium/utils.py (hunk 2 above)
Comment (Vaydheesh) [reply apertium/utils.py]: We do need to set LtLocale in apertium-lex-tools
https://github.com/apertium/apertium-lex-tools/blob/master/lrx_proc.cc#L100
I'll call the individual `Lt.Locale` Mayb we can have a single copy after all the files have been wrapped

File: apertium/utils.py [hunk 5]
```diff
@@ -27,32 +29,48 @@ def to_alpha3_code(code: str) -> str:
 
 def execute_pipeline(inp: str, commands: List[List[str]]) -> str:
     """
-    Args:
-        inp (str)
-        commands (List[List[str]])
+    Executes the given list of commands and returns the final output
 
     Returns:
```
Comment (sushain97) [inline apertium/utils.py]: IMO, it's pretty silly having these docstrings just repeat the type of the argument. Rather, they should describe the argument and then whatever document generation tool we use will automatically insert the type annotations (i.e. Sphinx). Just mentioning it now though. We can deal with it later.

**Pull request merged**